CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 #### _Black_
 
+- Refactor `src/black/__init__.py` into many files (#2206)
```

Comment:
not released yet, although will be soon!

```suggestion
### Unreleased

#### _Black_

- Refactor `src/black/__init__.py` into many files (#2206)

#### _Black_

```

Reply:
Thanks, fixed